apps/dashboard/src/app/team/[team_slug]/(team)/~/usage/overview/components/Usage.tsx

```diff
@@ -118,10 +136,25 @@ export const Usage: React.FC<UsageProps> = ({
       />
 
       <UsageCard
-        {...rpcMetrics}
-        name="RPC"
-        description="Amount of RPC requests allowed per second in your plan"
-      />
+        name="RPC Requests"
+        description={`Your plan allows for ${usageData.rateLimits.rpc} requests per second.`}
+      >
+        <ThirdwebBarChart
```

Comment:
Just pass the title and description to `ThirdwebBarChart` instead of putting it inside another card component

Reply:
let me push this change